core/src/main/java/org/elasticsearch/index/indexing/IndexingStats.java

```diff
@@ -155,6 +161,7 @@ public void readFrom(StreamInput in) throws IOException {
             indexCount = in.readVLong();
             indexTimeInMillis = in.readVLong();
             indexCurrent = in.readVLong();
+            indexFailedCount = in.readVLong();
```

Comment:
er - I think it goes like `if (in.version().onOrAfter(Version.V_2_1_0))` or something like that.